mumuki-domain.gemspec

```diff
@@ -14,7 +14,7 @@ Gem::Specification.new do |s|
 
   s.files = Dir["{app,db,lib}/**/*", "LICENSE", "Rakefile", "README.md"]
 
-  s.add_dependency "rails", "~> 5.1.6"
+  s.add_dependency "rails", "~> 5.2"
```

Comment:
Always use three digits with rails versions. It does not strictly follow semver, as there may be not-so-compatible changes between minor versions.